libvips/conversion/smartcrop.c

```diff
@@ -434,6 +452,21 @@ vips_smartcrop_class_init( VipsSmartcropClass *class )
 		G_STRUCT_OFFSET( VipsSmartcrop, interesting ),
 		VIPS_TYPE_INTERESTING, VIPS_INTERESTING_ATTENTION );
 
+	VIPS_ARG_INT( class, "attention_x", 2, 
+		_( "AttentionX" ), 
```

Comment:
This field is internationalised and is used as a label in eg. GUIs, so it needs to be one or two descriptive words of text with a leading capital letter. Perhaps `Attention x`? Sorry, I should have noticed this before.